### ISSUE TITLE
error all configured authentication methosds failed

### ISSUE THREAD

**Issue by enlinea777** [DockStation/dockstation#321]
the windows show this message error all configured authentication methods failed
but I am connected by ssh to the server with the private key
and when I try through the application it does not work

https://i.imgur.com/btbZX9N.png

my sshd_conf
```

Match User root 
	AllowTcpForwarding yes
	X11Forwarding yes
	PermitTunnel yes
	GatewayPorts yes
	AllowAgentForwarding yes
	PasswordAuthentication no
  PubkeyAuthentication yes 
```

**Comment by liucodeing** [comment]
https://github.com/zodern/meteor-up/issues/974

**Comment by jeff-h** [comment]
@liucodeing thanks so much for posting that! Finally I can use DockStation with my new VPS :) I must have spent *hours* trying to sort this out.

For others coming here, the solution is in https://github.com/zodern/meteor-up/issues/974#issuecomment-1124494397 specifically.

**Comment by enlinea777** [comment]
ok, It works very well

but also remember to configure the ssh server well in my case

```
PubkeyAcceptedKeyTypes=+ssh-rsa
PermitOpen any
```

With these new lines I already have remote access

**Comment by jeff-h** [comment]
Interesting, the only change I needed was to add the `PubkeyAcceptedKeyTypes=+ssh-rsa` line to my `/etc/ssh/sshd_config` file. It doesn't contain `PermitOpen any`.

**Comment by jeff-h** [comment]
I just fresh installed a new server with Ubuntu Server 22.04 and it had the exact same issue. The fix above worked flawlessly again here.

I wonder if this should be in the docs or on the front page perhaps?

**Comment by enlinea777** [comment]
exactly, it should be in a suggestion because most of us don't know.

> Acabo de instalar un nuevo servidor con Ubuntu Server 22.04 y tenía exactamente el mismo problema. La solución anterior funcionó perfectamente de nuevo aquí.
> 
> Me pregunto si esto debería estar en los documentos o quizás en la página principal.
